packages/docusaurus/src/server/routes.ts

```diff
@@ -136,7 +136,7 @@ export default async function loadRoutes(
     };
 
     return createRouteCodeString({
-      routePath: routeConfig.path,
+      routePath: routeConfig.path.replace(/'/g, "\\'"),
```

Comment:
was wondering, if using something like encodeURIComponent be a more general solution and handle other special chars?

Reply:
I don't think so, maybe it is worth resolving other cases when users will inform us about this?